types/topo_paths.go

```diff
@@ -33,20 +33,27 @@ var clabTmpDir = filepath.Join(os.TempDir(), ".clab")
 // generally all these paths are deduced from two main paths. The topology file path and the lab dir path.
 type TopoPaths struct {
 	topoFile           string
+	varsFile           string
 	labDir             string
 	topoName           string
 	externalCACertFile string // if an external CA certificate is used the path to the Cert file is stored here
 	externalCAKeyFile  string // if an external CA certificate is used the path to the Key file is stored here
 }
 
 // NewTopoPaths constructs a new TopoPaths instance.
-func NewTopoPaths(topologyFile string) (*TopoPaths, error) {
+func NewTopoPaths(topologyFile, varsFile string) (*TopoPaths, error) {
```

Comment:
rather than taking common.VarsFile shipping it straight into and storing it w/ the TopoPaths which just made sense to me